src/datepicker/datepicker-day-view.ts

```diff
@@ -0,0 +1,20 @@
+import {Component, Input} from '@angular/core';
+import {MonthViewModel, DayViewModel} from './datepicker-view-model';
+
+@Component({
+  selector: '[ngb-datepicker-day-view]',
```

Comment:
Why is this one using attribute as a selector? If we go down this path it should be camelCase instead of snake-case.